src/components/claiming/Countdown.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <NavigationCard
+    data-test="download-file"
+    title="It's done!"
+    :disabledNextButton="false"
+  >
+    <p>
+      You have completed the Witnet token claim process successfully on the
+      Sheikah side, but please
+      <span class="bold">REMEMBER TO SEND THE CLAIM FILE TO THIS ADDRESS:</span>
+    </p>
+    <a class="e-mail" href="mailto:genesis@witnet.foundation">
+      genesis@witnet.foundation
+    </a>
+    <p class="download-again">
+      If you lost your claim file, do not panic!
+      <span class="underline" @click="exportFile">Click here to download</span>
+      it again
+    </p>
+    <p class="text">
+      There is nothing to do on Sheikah until the genesis event takes place. In
+      the meanwhile, you are invited to join other community members in our
+      <a href="https://discord.gg/X4uurfP" target="_blank" class="underline">
+        Discord
+      </a>
+      and
+      <a href="https://t.me/witnetio" target="_blank" class="underline">
+        Telegram</a
+      >.
+    </p>
+    <div slot="alt-footer" class="countdown-background">
```

Comment:
this slot is not shown, the `NavigationCard` or the `BaseCard` must have lost info during the rebase.